website/content/docs/job-specification/resources.mdx

```diff
@@ -86,5 +86,50 @@ resources {
   }
 }
 ```
+## Memory Oversubscription
```

Comment:
FWIW, Not sure where this belongs. This seems the most relevant part - though, I considered having it in Internal/Scheduling documents.

Reply:
This location seems fine.